node/service/Cargo.toml

```diff
@@ -12,7 +12,7 @@ edition = "2021"
 targets = ["x86_64-unknown-linux-gnu"]
 
 [build-dependencies]
-substrate-build-script-utils = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.37" }
+substrate-build-script-utils = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.30" }
```

Comment:
Probably a typo ? Should be `branch = "polkadot-v0.9.40"` instead of `branch = "polkadot-v0.9.30"`.